tests/lang_es_test.rs

```diff
@@ -0,0 +1,88 @@
+use num2words::{Currency, Lang, Num2Err, Num2Words};
+use num_bigfloat::BigFloat;
+enum Outputs {
+    Cardinal,
+    Ordinal,
+    OrdinalNum,
+    Year,
+    Currency,
+}
+fn to_words(num: BigFloat, output: Outputs, preference: &[&str]) -> Result<String, Num2Err> {
```

Comment:
is there any reason why you are not using the actual builder each time, like on the other unit tests?

Reply:
two actually.
1-) I wanted to keep the `assert` line into 1 line if I could
2-) It was a bit easier to see what changed between one line and the other. (ngl, the Closure was overkill just to skip 1 parameter but welp...)